h/features/models.py

```diff
@@ -13,6 +13,8 @@
 
 FEATURES = {
     'activity_pages': "Show the new activity pages?",
+    'defer_realtime_updates': """Require a user action before applying real-time
+                                 updates to annotations in the client?""",
```

Comment:
Is `textwrap.dedent` the idiomatic way to remove the extra spacing in the string here? It doesn't make a visible difference because the whitespace is collapsed down to a single space when this string is rendered in the feature flags page of the admin control panel.